Makefile

```diff
@@ -34,8 +34,8 @@ PACKAGES := go list ./... | grep -v /vendor
 SOURCE_DIRS = cmd pkg test
 DOCS_SYNOPISIS_DIR = ./docs/source/_tmp
 
-$(GOPATH)/bin/minishift$(IS_EXE): $(BUILD_DIR)/$(GOOS)-$(GOARCH)/minishift$(IS_EXE)
-	cp $(BUILD_DIR)/$(GOOS)-$(GOARCH)/minishift$(IS_EXE) $(GOPATH)/bin/minishift$(IS_EXE)
+$(GOPATH)/bin/minishift$(IS_EXE): vendor $(shell $(GOFILES))
+	go install -ldflags="$(LDFLAGS)" ./cmd/minishift
```

Comment:
@hferentschik Not sure, what this line does. A few lines of brief explanation would help :smile: